tests/autoyast/repos.pm

```diff
@@ -30,7 +30,12 @@ use utils;
 sub run {
     # sles12_minimal.xml profile does not install "ip"
     assert_script_run 'ip a || ifstatus all';
-    quit_packagekit;
+    #quit_packagekit;
```

Comment:
Don't need delete this line? Just comment?

Reply:
Forgot to remove it, thanks.